cmd/cluster/support/post.go

```diff
@@ -163,6 +165,37 @@ func (p *Post) Run(clusterID string) error {
 	if err != nil {
 		return fmt.Errorf("can't retrieve cluster: %w", err)
 	}
+
+	subscriptionResponse, err := connection.
+		AccountsMgmt().
+		V1().
+		Subscriptions().
+		Subscription(p.cluster.Subscription().ID()).
+		Get().Send()
+	if err != nil {
+		return fmt.Errorf("failed to retrieve cluster subscription: %w", err)
+	}
+
+	labelResponse, err := connection.
+		AccountsMgmt().
+		V1().
+		Organizations().
+		Organization(subscriptionResponse.Body().OrganizationID()).
+		Labels().
+		Label(managedCriticalCustomerLabel).
+		Get().Send()
+	if err != nil {
+		// if the label is missing, there's no need to show an error to the user
+		if labelResponse.Error().Status() != http.StatusNotFound {
+			return fmt.Errorf("failed to retrieve cluster labels: %w", err)
+		}
+	} else if labelResponse.Body().Value() == "true" {
+		fmt.Println("This cluster is owned by a critical customer.")
```

Comment:
**Question:** Should we be more specific here in terms of what this means for the user?

Reply:
Great suggestion. What about something like this?
```
WARNING: This cluster is owned by a critical customer. Make sure that an SL has been sent and proactive case opened with the customer. Only continue if there has been no customer response for 24 hours.

See: https://source.redhat.com/groups/public/sre/wiki/defining_limited_support_process_for_osdrosa_for_critical_customers

Continue? (y/N)
```